test/harry/main/org/apache/cassandra/harry/model/PartitionState.java

```diff
@@ -223,7 +232,7 @@ private RowState updateRowState(RowState currentState, IntFunction<Bijections.Bi
 
                 assert lts >= currentState.lts[i] : String.format("Out-of-order LTS: %d. Max seen: %s", lts, currentState.lts[i]); // sanity check; we're iterating in lts order
 
-                if (currentState.lts[i] == lts)
+                if (lts != MagicConstants.NO_TIMESTAMP && currentState.lts[i] == lts)
```

Comment:
the `ast` tests down track timestamps so this would always hit this branch causing test failures... needed to disable this logic when doing a write and just assume that the write is *newer*

Reply:
"down track"?

Reply:
Sorry, bad sentence. 

the ast tests in this PR do not track timestamps, they rely on server to calculate them.  The existing logic expects harry to own the timestamps and update the `lts`, so i need a way to opt-out of this behavior; hence this change.

When the caller defines the `lts` as `NO_TIMESTAMP` that tells harry to just accept the write and override, rather than try to deal with timestamp conflicts